drivers/scheduler/oke/oke.go

```diff
@@ -577,6 +764,30 @@ func init() {
 	scheduler.Register(SchedName, o)
 }
 
+func (o *oke) getInstanceId(nodeName string) (string, error) {
+	cmd := fmt.Sprintf("%s compute instance list --compartment-id %s  --display-name %s --output json", ociCLI, o.compartmentId, nodeName)
+
+	stdout, stderr, err := osutils.ExecShell(cmd)
+	if err != nil {
+		return "", fmt.Errorf("failed to get instance list. stderr: %v, err: %v,stdout: %v", stderr, err, stdout)
+	}
+
+	instances := &Instance{}
+	err = json.Unmarshal([]byte(stdout), instances)
+	if err != nil {
+		return "", err
```

Comment:
Can we add some more verbose error here like you did above, I faced many issues in spawn due to OKE errors are not very straight forwardc (did a huge overhaul to add verbosity, which helped me track down some issues) and because we had very little prints in the functions, it was impossible to chase down where the error was actually coming from

Reply:
fixed it